src/metabase/util/schema.clj

```diff
@@ -399,6 +609,39 @@
                            s/Keyword                              s/Any}
     (deferred-tru "parameter must be a map with :id and :type keys")))
 
+(defschema ParameterSourceMalli
```

Comment:
This is here just because I want to try to use multi. 

Ideally, this should be part of `ParameterMalli` but I don't know how to merge a `[:map]` with a `[:multi]`. Suggestions are greatly appreciated.

Reply:
[this comment](https://github.com/metabase/metabase/pull/27471/files#r1059521556)